app/localization.js

```diff
@@ -31,7 +31,7 @@ export class Localizer {
             ? window.navigator.languages
             : [navigator.language || navigator.userLanguage];
 
-        for (let i = 0; i < userLanguages.length; i++) {
+        for (let i = 0; i < userLanguages.length; i += 1) {
```

Comment:
just because Crockford is afraid of the increment operator doesn't mean we should be too.  Please leave these in.

Reply:
agreed

Reply:
Sure.

I found `++i`, `i++` and `i +=1` all over the code. that's why I tried to normalize with one.

`++i` vs `i++` was making not so clear to me what the code was doing in some cases so I preferred `i += 1`.

In the case of for loops I agree that's irrelevant. But you can't tell eslint to enforce it everywhere except for for loops...